<launch>
	<arg name="mode" />

	<node pkg="multi_robot" type="obstacle_avoidance.py" name="obstacle_avoidance" args="--mode=$(arg mode)" output="screen"></node>


</launch>
